rdg_datasets/finance_fnma/README.md

```diff
@@ -0,0 +1,418 @@
+                                "PLAN_NUMBER": "string" 
+                            })
+cirt_node_properties = {
+    "Loan_CIRT": sf_node_properties["Loan_SF"],
+    "Borrower": sf_node_properties["Borrower"],
+    "Property": sf_node_properties["Property"],
+    "Loan_Snapshot": snapshot
+}
+
+nodes =  {
+    "Loan_SF": {
+        "id_column": "LOAN_ID", 
+        "id_space": "Loan_SF"
+    },
+    "Loan_CAS": {
+        "id_column": "LOAN_ID", 
+        "id_space": "Loan_CAS"
+    },    
+    "Loan_CIRT": {
+        "id_column": "LOAN_ID", 
+        "id_space": "Loan_CIRT"
+    },        
+    "Borrower" : {
+        "id_column": "BORROWER_ID", 
+        "id_space": "Borrower"
+    },
+    "Property": {
+        "id_column": "PROPERTY_ID", 
+        "id_space": "Property"
+    },
+    "Loan_Snapshot": {
+        "id_column": "LOAN_SNAPSHOT", 
+        "id_space": "Loan_Snapshot"
+    },
+    "MSA": {
+        "id_column": "MSA", 
+        "id_space": "MSA"
+    },
+    "Zip": {
+        "id_column": "ZIP", 
+        "id_space": "Zip"
+    },
+    "Seller": {
+        "id_column": "SELLER", 
+        "id_space": "SellerOrServicer"
+    },
+    "Servicer": {
+        "id_column": "SERVICER", 
+        "id_space": "SellerOrServicer"
+    },
+    "MBS": {
+        "id_column": "POOL_ID", 
+        "id_space": "MBS"
+    },
+    "Deal": {
+        "id_column": "DEAL_NAME", 
+        "id_space": "Deal"
+    }                                    
+}
+edges = {
+    "hasLoanSnapshot" : {
+        "source_label": "Loan",
+        "dest_label": "Loan_Snapshot"
+    },
+    "isCollateralizedBy" : {
+        "source_label": "Loan",
+        "dest_label": "Property"
+    },
+    "hasBorrower" : {
+        "source_label": "Loan",
+        "dest_label": "Borrower"
+    },
+    "isLocatedInMSA" : {
+        "source_label": "Property",
+        "dest_label": "MSA"
+    },
+    "isLocatedInZip" : {
+        "source_label": "Property",
+        "dest_label": "Zip"
+    },    
+    "originatedLoan" : {
+        "source_label": "Seller",
+        "dest_label": "Loan"
+    },
+    "servicesLoan" : {
+        "source_label": "Servicer",
+        "dest_label": "Loan"
+    },    
+    "hasSecuritizedLoan" : {
+        "source_label": "MBS",
+        "dest_label": "Loan"
+    },    
+    "hasLoan" : {
+        "source_label": "Deal",
+        "dest_label": "Loan"
+    }     
+}
+
+dataset_config = [
+    {
+        "dataset": "sf",
+        "xsm_path": "csv/finance_fnma/fnma_sf_2000Q1.csv",  
+        "loan_label": "Loan_SF",
+        "properties": sf_node_properties,
+        "nodes": ["Loan", "Borrower", "Property", "MSA", "Zip", "Loan_Snapshot", "Seller", "Servicer"],
+        "edges": ["hasLoanSnapshot", "hasBorrower", "isCollateralizedBy", "isLocatedInMSA", "isLocatedInZip", "originatedLoan", "servicesLoan"]
+    } 
+]
+
+graph = remote.Client(disable_version_check=False).create_graph(
+    num_partitions=4
```

Comment:
if we don't need it partitioned, please update the `num_partitions` here

Reply:
We do need partitioning because of the limitations with Dask importer. Dask importer requires minimum 3 hosts. There should be no issue importing a partitioned RDG on 1 host, is there another reason why we need to have an unpartitioned RDG?

Reply:
> There should be no issue importing a partitioned RDG on 1 host

the open source katana code cannot load a partitioned RDG at all

> is there another reason why we need to have an unpartitioned RDG?

it wastes CI/testing time to have to repartition the RDG every time we load it, if we really just want an unpartitioned RDG (which it sounds like we do) we should store an unpartitioned RDG


Could you please repartition this RDG using M:N partitioning?

Reply:
Yes will do

Reply:
Re-partitioned to 1 partition